EmailNotificationActivity/__tests__/handler.test.ts

```diff
@@ -0,0 +1,165 @@
+/* tslint:disable:no-any */
+/* tslint:disable:no-duplicate-string */
+/* tslint:disable:no-big-function */
+/* tslint:disable: no-identical-functions */
+
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "italia-ts-commons/lib/strings";
+
+import {
+  EmailNotificationActivityInput,
+  getEmailNotificationActivityHandler
+} from "../handler";
+
+import { some } from "fp-ts/lib/Option";
+import { fromLeft, taskEither } from "fp-ts/lib/TaskEither";
+
+import { EmailAddress } from "io-functions-commons/dist/generated/definitions/EmailAddress";
+import { MessageBodyMarkdown } from "io-functions-commons/dist/generated/definitions/MessageBodyMarkdown";
+import { MessageContent } from "io-functions-commons/dist/generated/definitions/MessageContent";
+import { MessageSubject } from "io-functions-commons/dist/generated/definitions/MessageSubject";
+import { TimeToLiveSeconds } from "io-functions-commons/dist/generated/definitions/TimeToLiveSeconds";
+
+import { NotificationChannelEnum } from "io-functions-commons/dist/generated/definitions/NotificationChannel";
+import * as mail from "io-functions-commons/dist/src/mailer";
+import { CreatedMessageEventSenderMetadata } from "io-functions-commons/dist/src/models/created_message_sender_metadata";
+import {
+  NewNotification,
+  NotificationAddressSourceEnum,
+  NotificationModel,
+  RetrievedNotification
+} from "io-functions-commons/dist/src/models/notification";
+
+describe("getEmailNotificationActivityHandler", () => {
+  const mockContext = {
```

Comment:
Add this just to be sure tests are isolated (mocked values and implementations persist for all tests in a file)
```suggestion
  beforeEach(() => jest.clearAllMocks());
  const mockContext = {
```